frontend/src/screens/MyPromotions.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { CSSProperties, ReactElement } from "react";
+import { Checkbox, Row, Col } from 'antd';
+
+import DropdownMenu from "../components/DropdownMenu";
+import PromotionCard from "../components/promotion/PromotionCard";
+import UploadPromoButton from "../components/button/UploadPromoButton";
+
+import { DropdownOption, Dropdown} from "../types/Dropdown";
+import { Promotion } from "../types/Promotion";
+
+const options: DropdownOption[] = [
+  {
+    link: "/link1",
+    text: "Option 1",
+  },
+  {
+    link: "/link2",
+    text: "Option 2",
+  },
+  {
+    link: "/link3",
+    text: "Option 3",
+  },
+];
+
+const dropdowns: Dropdown[] = [
+  {
+    text: "Sort",
+    options,
+  },
+  {
+    text: "Category",
+    options,
+  },
+];
+
+const promotions: Promotion[] = [
+  {
+    title: "Happy Hour 2pm-4pm",
+    restaurantName: "Starbucks",
+    description: "Buy one, get one free on all Grande sized drinks!",
+    date: "Nov 11, 2020",
+    liked: false,
+    image: { src: "" },
```

Comment:
Hmm do you want some mock data that's accurate to what we have in the BE? It'll make it a bit easier for you so you won't need to make as many changes once the endpoint PR is pushed.
[SamplePromotionDate.txt](https://github.com/ubclaunchpad/foodies/files/5540276/SamplePromotionDate.txt)

Reply:
Let me know if you want any user sample data, but the BE endpoints for both promotions and users should be merged relatively soon

Reply:
Thanks for the sample data! Do you think we should be filtering and formatting the specific data we need (ie. for a PromotionCard) in the frontend or the backend?